files/private-chef-cookbooks/private-chef/templates/default/sv-bookshelf-run.erb

```diff
@@ -1,4 +1,11 @@
-#!/bin/sh
+#!/bin/bash
 exec 2>&1
 
+source 'sv_lib'
+
+ensure_backing_storage
+if [[ $? != 0 ]]; then
```

Comment:
`set -e` was just a thought as I was reading through the code, let's not do that.

As an aside though, if we end up having runit scripts that do tons of conditional cleanup, I'm going to cry a lot.